src/test/java/org/jabref/logic/bibtexkeypattern/MakeLabelWithoutDatabaseTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jabref.logic.bibtexkeypattern;
+
+import org.jabref.model.entry.BibEntry;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class MakeLabelWithoutDatabaseTest {
+
+    // private GlobalBibtexKeyPattern pattern;
+    private BibEntry entry;
+
+    @Before
+    public void setUp() {
+        entry = new BibEntry();
+        entry.setField("author", "John Doe");
+        entry.setField("year", "2016");
+        entry.setField("title", "An awesome paper on JabRef");
+    }
+
+    @Test
+    public void makeLabelForFileSearch() {
+        String label = BibtexKeyPatternUtil.makeLabel(entry,
+                /*value=*/ "auth",
```

Comment:
I think we never used comments to indicate the name of the argument and thus would prefer that you remove them.